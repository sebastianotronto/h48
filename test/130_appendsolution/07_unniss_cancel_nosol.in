B' F
R2 B2
1
1
rotation UF
