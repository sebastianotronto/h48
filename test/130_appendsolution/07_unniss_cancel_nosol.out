Number of solutions: 0
Shortest solution length: 21
Used bytes: 0
